app/[lang]/error.tsx

```diff
@@ -22,30 +22,23 @@ export default function Error({
   }, [error])
 
   return (
-    <div className="flex h-full grow items-end justify-center">
+    <div className="flex h-screen flex-col justify-center">
       <Image
-        src="/assets/images/404.jpg"
+        src="/assets/images/error-page_cat.jpg"
         alt="page not found"
-        fill
-        quality={100}
-        loading="eager"
-        priority
-        className="-z-20 h-full w-full object-cover"
+        width={250}
+        height={250}
+        className="flex self-center"
       />
-
-      <Topbar />
-      <div className="absolute top-1/2 left-1/2 z-0 flex w-screen -translate-x-1/2 -translate-y-1/2 flex-col justify-center">
+      <div className="flex flex-col justify-center">
         <h1 className=" text-center font-cbrush text-6xl text-white">
           {t('error.first')}
         </h1>
         <h1 className="text-center font-cbrush text-4xl text-white">
```

Comment:
Same here for the font sizing lets make the h1 be `text-5xl and md:text-6xl` and the h2 be `text-2xl md:text-4xl`